boards/arm/reel_board/board.c

```diff
@@ -37,6 +37,11 @@ static int board_reel_board_init(struct device *dev)
 		(GPIO_PIN_CNF_INPUT_Disconnect << GPIO_PIN_CNF_INPUT_Pos) |
 		(GPIO_PIN_CNF_PULL_Pullup << GPIO_PIN_CNF_PULL_Pos) |
 		(GPIO_PIN_CNF_DIR_Input << GPIO_PIN_CNF_DIR_Pos);
+	gpio->PIN_CNF[DT_NORDIC_NRF_UART_0_TX_PIN] =
+		(GPIO_PIN_CNF_INPUT_Disconnect << GPIO_PIN_CNF_INPUT_Pos) |
+		(GPIO_PIN_CNF_PULL_Pullup << GPIO_PIN_CNF_PULL_Pos) |
+		(GPIO_PIN_CNF_DIR_Input << GPIO_PIN_CNF_DIR_Pos);
```

Comment:
Perhaps it would be good to mention in the comment above that the same configuration is used for the TX pin, for the same reason. One may find it surprising that the TX pin is configured as an input.